target-platform/target-platform.target

```diff
@@ -4,18 +4,18 @@
 	<targetJRE path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-17"/>
 	<locations>
 		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
-			<repository location="https://download.eclipse.org/eclipse/updates/4.23/"/><!--Keep in sync with repo-ref in org.eclipse.m2e.repository/category.xml-->
+			<repository location="https://download.eclipse.org/eclipse/updates/4.24-I-builds/"/><!--Keep in sync with repo-ref in org.eclipse.m2e.repository/category.xml-->
```

Comment:
This should be changed to the final address `https://download.eclipse.org/eclipse/updates/4.24` as soon as it is available, which should be today. So I think we can wait for that.